src/locales/en.json

```diff
@@ -39,17 +39,17 @@
     "imprint": "Imprint",
     "loggingin": "Logging in...",
     "logout": "@:actions.logout",
-    "mainTitle": "Your flexible time sheet",
+    "mainTitle": "Your flexible timesheet",
     "news": "News",
     "notfound": "Not found",
     "onboarding": "Onboarding",
     "privacy": "Privacy",
     "privacyagreement": "Privacy agreement",
-    "reports": "Report",
+    "reports": "timesheets",
```

Comment:
sollten wir das dann nicht auch groß schreiben?

Reply:
yes. Wobei es manchmal Fälle gibt, in denen ein Begriff injected wird und dann in einem Satz steht. Spielt aber hier so keine große Rolle, denke ich.